stream_alert_cli/config.py

```diff
@@ -81,105 +39,67 @@ def __getitem__(self, key):
         return self.config[key]
 
     def __setitem__(self, key, new_value):
-        if key in self.v2_schema:
-            self.config.__setitem__(key, new_value)
-            self.write()
-        else:
-            raise ConfigError('Invalid key to set: {}'.format(key))
+        self.config.__setitem__(key, new_value)
+        self.write()
 
     def get(self, key):
         return self.config.get(key)
 
+    def clusters(self):
+        return self.config['clusters'].keys()
+
     def load(self):
-        """Load the variables.json configuration file
+        """Load the cluster, global, and lambda configuration files
 
         Returns:
-            [dict] loaded config from variables.json
+            [dict] loaded config from all config files with the following keys:
+                'clusters', 'global', and 'lambda'
         """
-        if not os.path.isfile(self.filename):
-            raise ConfigError('StreamAlert variables.json file not found!')
 
-        with open(self.filename) as data:
-            try:
-                config = json.load(data)
-            except ValueError:
-                raise ConfigError('StreamAlert variables.json file is not valid JSON!')
-            return config
+        config = {}
+        config['clusters'] = {}
 
-    def write(self):
-        """Write the current config in memory to disk"""
-        with open(self.filename, 'r+') as varfile:
-            varfile.write(json.dumps(
-                self.config,
-                indent=4,
-                separators=(',', ': '),
-                sort_keys=True
-            ))
-            varfile.truncate()
-
-    def _detect_version(self):
-        """Detect the config version
-
-        Returns:
-            [int] detected config version
-        """
-        config_keys = set(self.config.keys())
+        def _config_loader(key, filepath, cluster_file):
+            if not os.path.isfile(filepath):
+                LOGGER_CLI.error('%s not found!', filepath)
```

Comment:
Make this error more descriptive - ie, mention the operation (config loading). also quit using so many exclamation points. this is not exciting.